VTUL/dlp-access
# Upgrade npm packages

**JIRA Ticket**: (https://webapps.es.vt.edu/jira/browse/LIBTD-2118) (:star:)

# What does this Pull Request do? (:star:)
Fix npm vulnerabilities from 38935 to 1

# How should this be tested?
1. Verify results in Amplify console
2. Test site functionalities 
3. Local test by launching the Cypress app ```yarn run cypress open```

# Interested parties
Tag (@tingtingjh, @whunter, @andreaWaldren  ) interested parties

(:star:) Required fields
